### DISCUSSION TITLE
application状态

### DISCUSSION THREAD

**Discussion by EMinEMh** [kubevela/kubevela#3443] [Q&A] [answered]
我发现apply的application是running状态，而且组件health为true，但是我describe 组件时发现组件的运行状态和我在componet定义的健康状态检查条件是不一致的

% kubectl get app 4788595374973a2b -n cac-dev-env         
NAME               COMPONENT                     TYPE                     PHASE     HEALTHY   STATUS   AGE
4788595374973a2b   4788595374973a2b-deployment   component-deployment   running   true               13m

% kubectl describe app 4788595374973a2b -n cac-dev-env (截取service段)
 Services:
    Healthy:  true
    Name:     4788595374973a2b-deployment
    Workload Definition:
      API Version:  cac.test.com/v1beta1
      Kind:         Deployment
    Healthy:        true
    Name:           4788595374973a2b-dcloud
    Traits:
      Healthy:  true
      Type:     trait-dscaler
      Healthy:  true
      Type:     trait-dnetworkconf
    Workload Definition:
      API Version:  cac.test.com/v1beta1
      Kind:         Dcloud

% kubectl describe Dcloud 4788595374973a2b-dcloud -n cac-dev-env (截取status段)
Status:
  Id:      1246
  Reason:  
  Status:  process-updating
Events:    <none>

而我设置的component-dcloud（Dcloud）健康条件为：
status:
      healthPolicy: |
        isHealth: context.output.status.status == "ready"

vela版本：v1.2.4

请问这会是什么原因导致？

**Comment by EMinEMh** [comment, marked as answer]
已解决，自身配置问题

**Reply by wangqi811** [reply]
我好像也碰到这个问题了，请问下是怎么解决的

**Reply by EMinEMh** [reply]
不好意思刚看到，之前是我配置原因导致vela不断重启，所以导致状态不一致
